packages/repeater/src/lib/Repeater.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  ExecuteRequestEventHandler,
+  RegisterRepeaterCommand,
+  RegisterRepeaterResult,
+  RepeaterStatusEvent
+} from '../bus';
+import { RepeaterStatus } from '../models';
+import { Configuration, EventBus } from '@secbox/core';
+import Timer = NodeJS.Timer;
+
+export class Repeater {
+  public readonly repeaterId: string;
+  private readonly bus: EventBus;
+  private readonly configuration: Configuration;
+
+  private timer?: Timer;
+
+  constructor({
+    repeaterId,
+    bus,
+    configuration
+  }: {
+    repeaterId: string;
+    bus: EventBus;
+    configuration: Configuration;
+  }) {
+    this.repeaterId = repeaterId;
+    this.bus = bus;
+    this.configuration = configuration;
+  }
+
+  public async start(): Promise<void> {
+    await this.register();
```

Comment:
We should verify the result to make sure that a repeater has been registered, otherwise just raise an exception